pandas/tests/indexes/test_common.py

```diff
@@ -337,3 +337,25 @@ def test_has_duplicates(self, indices):
         idx = holder([indices[0]] * 5)
         assert idx.is_unique is False
         assert idx.has_duplicates is True
+
+    @pytest.mark.parametrize("dtype", ["int64", "uint64", "float64", "category"])
```

Comment:
was this the one where adding more dtypes here led to 90k tests?

Reply:
I don't think it's too many; added a couple more. Let me know if there are others you think I should include.